package.json

```diff
@@ -108,6 +112,8 @@
     "axios-mock-adapter": "^1.21.1",
     "babel-jest": "^28.1.2",
     "babel-loader": "^8.2.4",
+    "chalk": "^4.1.2",
+    "child_process": "^1.0.2",
```

Comment:
[curious] I have a hunch these dependencies will need to be installed of regular `dependencies`, including `inquirer` below.

If installed as `devDependencies`, their code won't be included in the installed bundle in the consuming application, which could lead to not being able to run the Paragon CLI correctly. It's not a concern within the `example` app in the Paragon repo itself, because these packages are installed into the root `node_modules` via the `devDependencies`. However, for MFEs outside of the Paragon repo, this could be a concern.

I wonder if it's worth a sanity check of trying to run the Paragon CLI outside of the `example` MFE app, too? Perhaps installing this branch of Paragon into `frontend-template-application` would be a good test (I _think_ the `bin` script should work?). 

`npm install "https://github.com/openedx/paragon.git#raccoongang:zadorozhnii/paragon-cli-for-theming" --save`

Note this also related to the `// eslint-disable-next-line import/no-extraneous-dependencies` comments.